recipes/libpsl/all/conanfile.py

```diff
@@ -48,18 +46,18 @@ def layout(self):
 
     def requirements(self):
         if self.options.with_idna == "icu":
-            self.requires("icu/73.2")
+            self.requires("icu/74.1")
         elif self.options.with_idna == "libidn":
             self.requires("libidn/1.36")
         elif self.options.with_idna == "libidn2":
             self.requires("libidn2/2.3.0")
         if self.options.with_idna in ("libidn", "libidn2"):
-            self.requires("libunistring/0.9.10")
+            self.requires("libunistring/1.1")
```

Comment:
Podofo is the only library that uses this too, we should bring it in-line with this in a follow-up PR @danimtb @valgur